plugins/modules/remote_management/redfish/wdc_redfish_command.py

```diff
@@ -191,7 +203,9 @@
     ],
     "Chassis": [
         "IndicatorLedOn",
-        "IndicatorLedOff"
+        "IndicatorLedOff",
+        "PowerModeLow",
+        "PowerModeNormal"
```

Comment:
This will reduce the diff size the next time you add something:
```suggestion
        "PowerModeNormal",
```